src/components/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import {useHistory} from "react-router-dom";
```

Comment:
```suggestion
import { useHistory } from "react-router-dom";
```

налаштуй прітіер так, шоб він робив спейс між дужками

@TaniaMitnovych  @NazarMykhailiv @Yurii-Kupchynskyi  ви теж, будь ласка